### PR TITLE
Add godox to .travis.yml

### PR DESCRIPTION
**Pull request by Sean-Der**
Eventually this will be added to golangci-lint, but for now
we have to do it manually

Relates to #106

### CODE REVIEW

File: datachannel.go [hunk 1]
```diff
@@ -140,7 +140,7 @@ func (d *DataChannel) open(sctpTransport *SCTPTransport) error {
 
 	cfg := &datachannel.Config{
 		ChannelType:          channelType,
-		Priority:             datachannel.ChannelPriorityNormal, // TODO: Wiring
```
Comment (Sean-Der) [inline datachannel.go]: This seems to be respected/fully implemented?

File: peerconnection.go [hunk 2]
```diff
@@ -1567,27 +1573,15 @@ func (pc *PeerConnection) Close() error {
 	pc.signalingState = SignalingStateClosed
 
 	// https://www.w3.org/TR/webrtc/#dom-rtcpeerconnection-close (step #11)
-	// pc.ICEConnectionState = ICEConnectionStateClosed
-	pc.iceStateChange(ice.ConnectionStateClosed) // FIXME REMOVE
```
Comment (Sean-Der) [inline peerconnection.go]: We don't need to set this ourselves `pion/ice` sets this to Closed for us

**Comment by Sean-Der** [comment]
@trivigy @albrow @backkem @enobufs 

I am moving all our `TODO` comments into real bugs. I have noticed that most of our TODOs don't get worked (until someone hits a bug related to them!)

This PR is going to take a long time to finish, but would you mind looking through the commits and signing off on what I have now? I will cherry-pick to master, this is probably going to be a week long process.

File: rtptransceiver.go [hunk 3]
```diff
@@ -8,7 +8,6 @@ import (
 
 // RTPTransceiver represents a combination of an RTPSender and an RTPReceiver that share a common mid.
 type RTPTransceiver struct {
-	Mid       string
```
Comment (Sean-Der) [inline rtptransceiver.go]: See https://github.com/pion/webrtc/pull/745/commits/5492a221456a8bdd4dce014d0512bd0c3bd0de0c for why this was removed

File: peerconnection.go (hunk 2 above)
Comment (Sean-Der) [reply peerconnection.go]: I also added a test that confirms that `Closed` is properly set. Users should see no behavior difference

File: .travis.yml [hunk 4]
```diff
@@ -21,6 +21,7 @@ install:
   - export PATH=$GOROOT/bin:$PATH
   - go version
   - go env
+  - go get -u github.com/matoous/godox
```
Comment (Sean-Der) [inline .travis.yml]: `godox` may be added to `golangci-lint`

If not I might just extend our current `panic` finder to look for `TODO`, `FIXME` and `BUG`

I think this is the right decision. Currently people contribute code, but don't drive the tech debt down. This is also a good way to get things in the bug tracker so others can be aware/help out!

**Comment by trivigy** [comment]
@Sean-Der You think you could separate the MID work from the TODO stuff? Will be easier to reverse later in case of debugging.

**Comment by Sean-Der** [comment]
Done! @trivigy https://github.com/pion/webrtc/pull/745/commits/48963f5458503063a609f27ffb44225bf9cb9f67 put it in its own commit

**Pull request merged**